### DISCUSSION TITLE
How do I do multiline buttons, and datum question.

### DISCUSSION THREAD

**Discussion by trailsurfer604** [Bodmer/TFT_eSPI#2268] [Q&A - General] [answered]
Hi,

Two questions:

1. Is it possible to do multiline buttons? (i.e. buttons where the label is on two lines.
I tried 
```
char * MENU_BUTTON_TEXT_OFF[16] = {
  "Boost\nDigital",        "EGT\nDigital",
  "Coolant Digital",      "Trans Digital", 
  "Day Mode",             "Exit"
};
```
but the \n simply gets swallowed.

2. Can someone please point me to a source that explains what datum (as in setFontDatum) is and how it works?

**Comment by Bodmer** [comment, marked as answer]
1. No, only single line buttons are supported. As a work around you could create two buttons in the same location.
2. See [example](https://github.com/Bodmer/TFT_eSPI/blob/master/examples/480%20x%20320/TFT_String_Align/TFT_String_Align.ino).
